packages/core/src/App/AppContent.tsx

```diff
@@ -68,9 +69,15 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
     const is_app_id_set = localStorage.getItem('config.app_id');
     const is_change_login_app_id_set = localStorage.getItem('change_login_app_id');
 
+    // use prompt=none silent login checks for Chrome and Firefox
     useSilentLoginAndLogout({
         is_client_store_initialized,
         isOAuth2Enabled,
+    });
+    // use logged_state cookie login checks for Safari
+    useLoggedStateLoginAndLogout({
+        is_client_store_initialized,
+        isOAuth2Enabled,
```

Comment:
This is to ensure Safari is still using `logged_state`, because at the moment front channels do not work in Safari due to Storage partitioning in [Intelligent Tracking Prevention](https://webkit.org/tracking-prevention/)
![Screenshot 2025-02-21 at 4 13 17 PM](https://github.com/user-attachments/assets/f7a3233f-1211-4374-82ca-7672bcdd4a69)


https://stackoverflow.com/questions/63921299/safari-setting-localstorage-inside-iframe-is-not-correctly-shared

Reply:
So if we log out at `https://hub.deriv.com`, Hydra will render front channels to `https://app.deriv.com/front-channel.html` which SHOULD clear local storage in Chrome and Firefox, but it will not work in Safari due to the issue mentioned above